server/app/models/user.rb

```diff
@@ -16,6 +16,8 @@ def as_json(options={})
       super(options)
     end
 
+  has_and_belongs_to_many :companies
```

Comment:
I think we want a user to 'belong to' a company - I don't think we want a user to be able to have multiple companies, right?